deployments/utoronto/image/Dockerfile

```
@@ -42,18 +42,20 @@ RUN echo "${LC_ALL} UTF-8" > /etc/locale.gen && \
 RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
 RUN echo "deb https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/" > /etc/apt/sources.list.d/cran.list
 
+
 # Install R packages
 # Our pre-built R packages from rspm are built against system libs in focal
 # rstan takes forever to compile from source, and needs libnodejs
-# So we install older (10.x) nodejs from apt rather than newer from conda
+# We don't want R 4.1 yet - the graphics protocol version it has is incompatible
+# with the version of RStudio we use. So we pin R to 4.0.5
+ENV R_VERSION=4.0.5-1.2004.0
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
+RUN echo "deb https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/" > /etc/apt/sources.list.d/cran.list
 RUN apt-get update -qq --yes > /dev/null && \
     apt-get install --yes -qq \
-    r-base \
-    r-base-dev \
-    r-recommended \
-    r-cran-littler \
-    nodejs \
-    npm
+    r-base-core=${R_VERSION} \
+    r-base-dev=${R_VERSION} \
+    r-cran-littler=0.3.11-1.2004.0 > /dev/null
```

The image build CI seems to be complaining about these two missing

```suggestion
    r-cran-littler=0.3.11-1.2004.0 \
    nodejs \
    npm > /dev/null
```

---

Great catch!